lib/ace/lib/es5-shim.js

```diff
@@ -984,10 +984,10 @@ if (!Date.now) {
 
 // ES5 15.5.4.20
 // http://es5.github.com/#x15.5.4.20
-var ws = "\x09\x0A\x0B\x0C\x0D\x20\xA0\u1680\u180E\u2000\u2001\u2002\u2003" +
```

Comment:
why do we need to remove `\u180E`?

Reply:
It is not included in the current version of the spec, so was causing us to override the builtin version of trim in all browsers with our implementation